.gitignore

```diff
@@ -4,11 +4,15 @@
 
 # Eclipse, Netbeans and IntelliJ files
 /.*
-!.gitignore
+.gitignore
```

Comment:
Should we really ignore the `.gitignore` file? I think this file is supposed to be committed into the source repo.